crypto/bio/bio_local.h

```diff
@@ -27,10 +27,10 @@
  * For clarity, we check for internal/cryptlib.h since it's a common header
  * that also includes bio.h.
  */
-# ifdef HEADER_CRYPTLIB_H
+# ifdef OPENSSL_INTERNAL_CRYPTLIB_H
```

Comment:
TODO: here are some more external include guards which need to be checked.

Reply:
This one (and the following) is ok, it was added for safety in 28a0841bf58 and [serves a useful purpose](https://github.com/openssl/openssl/commit/28a0841bf58e3813b2e07ad22f19484308e2f70a#diff-ec439abd0c9ef34a47a04923dbf2bc2fR15).